pkg/serve/run_handler.go

```diff
@@ -154,6 +171,46 @@ func RunHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func cpuUtilizations(dbFileName string, scenarioRunId int64) []CPUUtilizationMetric {
+	totalConn, err := sqlite3.Open(dbFileName, sqlite3.OPEN_READONLY)
+	if err != nil {
+		panic(fmt.Errorf("could not open database file '%s': %s", dbFileName, err.Error()))
+	}
+	defer totalConn.Close()
+
+	cpuUtilizationStmt, err := totalConn.Prepare(data.CPUUtilizationQuery, scenarioRunId)
```

Comment:
At the time I chose to centralise the queries because it makes them easier to work with in an IDE. Or at least, I felt so. My SQL background / bias showing.